website/server/libs/payments.js

```diff
@@ -289,7 +435,7 @@ api.cancelSubscription = async function cancelSubscription (data) {
     await data.user.save();
   }
 
-  txnEmail(data.user, emailType);
+  if (customerId !== this.constants.GROUP_PLAN_CUSTOMER_ID) txnEmail(data.user, emailType, emailMergeData);
```

Comment:
No, the email is only sent if the plan is not a user group plan (the free plan that all users get if they are members of a group with a group plan)